charts/matrix-stack/templates/matrix-rtc/_helpers.tpl

```diff
@@ -121,10 +121,40 @@ app.kubernetes.io/version: {{ .image.tag }}
 {{- end }}
 
 
-{{- define "element-io.matrix-rtc-sfu.config" }}
+{{- define "element-io.matrix-rtc-sfu.configmap-data" }}
 {{- $root := .root -}}
 {{- with required "element-io.matrix-rtc-sfu.config missing context" .context -}}
 {{- $config := (tpl ($root.Files.Get "configs/matrix-rtc/sfu/config.yaml.tpl") (dict "root" $root "context" .)) | fromYaml }}
-{{- toYaml (merge (.additional | fromYaml) $config) }}
+config.yaml: |
+  {{- toYaml (merge (.additional | fromYaml) $config) | nindent 2 }}
+{{- if not ($root.Values.matrixRTC.livekitAuth).keysYaml }}
+keys-template.yaml: |
+  ${LIVEKIT_KEY}: ${LIVEKIT_SECRET}
+{{- end -}}
```

Comment:
These 2 feel so simple that it isn't worth putting in an included file. But consistency?